explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -645,15 +645,15 @@ export const buildTripleColumnGrid = (): React.ComponentProps<
  */
 function calculateEstimatedCellCount(
   projectsResponse: ProjectsResponse
-): string {
+): number | null {
   const estimatedCellCount =
     getProjectResponse(projectsResponse).estimatedCellCount;
   // If there's an estimated cell count for the project, return it as the cell count.
   if (estimatedCellCount) {
-    return estimatedCellCount.toLocaleString();
+    return estimatedCellCount;
   }
   // Otherwise, return the cell suspension total count.
-  return getCellSuspensionTotalCells(projectsResponse);
+  return rollUpTotalCells(projectsResponse);
```

Comment:
`rollUpTotalCells` is the returned calculated value of `totalCells` in the function `getCellSuspensionTotalCells` - so essentially we have removed the formatting of this value.